tests/test_api/test_endpoints.py

```diff
@@ -964,10 +963,39 @@ def test_run_query_aggregate_with_continuation(api_user_client):
                     }
                 },
             ],
+            # TODO: Why is this "lever" (whether the MongoDB server can use a disk) being exposed to the API client?
+            #       Also, according to the MongoDB docs, this is effectively `True` by default:
+            #       > "[...] pipeline stages that require more than 100 megabytes of memory to execute
+            #       > write temporary files to disk by default."
+            #       Reference: https://www.mongodb.com/docs/manual/reference/method/cursor.allowDiskUse
+            #
```

Comment:
Excellent. Yes, this is enabled by default now (c.f. <https://www.mongodb.com/docs/manual/reference/parameters/#mongodb-parameter-param.allowDiskUseByDefault>. So, we can remove this user-settable parameter from our interface.

Reply:
@eecavanna the "why" is that I've been using mongodb since version 3, and I don't always catch the improvements in default values. 😄